code/debugconsole/console.h

```diff
@@ -119,27 +119,6 @@ class debug_command;
 		}	\
 	}
 
- /**
-  * @def Shortcut for single-variable setters/monitors
-  *
-  * @param [in] function_name
-  * @param [in] float_variable
-  * @param [in] short_help
-  */
- #define DCF_FLOAT(function_name, float_variable, short_help)	\
```

Comment:
This should not be removed since it might be useful later.

Reply:
done